ms_office_macro_bleach/bleach.py

```diff
@@ -0,0 +1,76 @@
+"""This module removes any and all macros/dynamic content from MS Office files.
+
+It is part of a suite of programs developed by the AntiMalware Alliance.
+
+Visit https://github.com/Anti-Malware-Alliance for more details about our organisation and projects.
+
+Feel free to contact benjamin.mcgregor2002@gmail.com for any questions regarding this module.
+"""
+from argparse import ArgumentParser
+from os import rename, path, remove
+from zipfile import ZipFile
+from shutil import make_archive, rmtree
+
+
+# Unzip file function
+# Converts file to a .zip archive and extracts it to file directory
+def unzip_file(file):
+
```

Comment:
Before unziping is probably a good idea to look into filesize and define an upper limit to filesize, to avoid an attack where a file grows exponentially large and you run out of space. Doing something like 200MB is probably a safe start.